### DISCUSSION TITLE
I2S DAC for analog X,Y - of any use?

### DISCUSSION THREAD

**Discussion by terjeio** [terjeio/grblHAL#138] [Ideas]
Added to STM32F446 driver via [board code](https://github.com/terjeio/grblHAL/blob/test/drivers/STM32F4xx/Src/st_morpo_dac.c) only, PCM5100 32-bit stereo audio DAC, DMA - done for fun and as an example of how "hackable" grblHAL is:

![grblHAL](https://user-images.githubusercontent.com/20260062/101954633-b5091200-3bfc-11eb-85cc-da4a1951cfa8.png)

Logo is by @phil-barrett - a bit misrepresented since I did not connect a Z-input to my scope (spindle output can be used in laser mode) so rapids are plotted when they should not. Laser galvo?

**Comment by phil-barrett** [comment]
I like it. Kind of an Etch A Sketch approach.  Submitted it to HaD yet?

**Reply by terjeio** [reply]
Nope - just plotted it. My idea was to make something for the christmas challenge but don't know if it is too late for that...

**Comment by phil-barrett** [comment]
Given the completely lame stuff they have been running lately, they will probably put yours up, even if late.
